src/Raven.Studio/typescript/viewmodels/database/documents/compareExchangeWarningsConfirm.ts

```diff
@@ -0,0 +1,51 @@
+import dialog = require("plugins/dialog");
+import dialogViewModelBase = require("viewmodels/dialogViewModelBase");
+
+class compareExchangeWarningsConfirm extends dialogViewModelBase {
```

Comment:
I think we already have such dialog - maybe rework existing dialog with support for type and list of annotations? Then in goto callback we can pass both type and annotation?

Reply:
done